src/icons/fileIcons.ts

```diff
@@ -1597,7 +1598,27 @@ export const fileIcons: FileIcons = {
       fileNames: ['makefile', 'gnumakefile', 'kbuild'],
     },
     { name: 'foxpro', fileExtensions: ['fxp', 'prg'] },
-    { name: 'i18n', fileExtensions: ['pot', 'po', 'mo', 'lang'] },
+    {
+      name: 'i18n',
+      fileExtensions: [
+        'pot',
+        'po',
+        'mo',
+        'lang',
+        'nls.json',
```

Comment:
consider using a parent repository pattern instead

Reply:
What do you mean?

Reply:
I mean if these files usually appear inside a "i18n" folder, you can use this instead. But i remember that this is usually found at the root folder, so it might not be feasible